integration/console/schematic/button.py

```
@@ -0,0 +1,129 @@
+            applied_properties["show_control_chip"] = show_control_chip
+
+        if mode is not None:
+            # Convert to title case for UI interaction
+            mode_title = mode.title()
+            self.page.get_by_text(mode_title, exact=True).click()
+            applied_properties["mode"] = mode.lower()
+
+        return applied_properties
+
+    def get_properties(self) -> Dict[str, Any]:
+        """Get the current properties of the symbol"""
+        self._click_symbol()
+        self.page.get_by_text("Properties").click()
+        self.page.get_by_text("Control").last.click()
+
+        props: Dict[str, Any] = {
+            "channel": "",
+            "activation_delay": -1.0,
+            "show_control_chip": False,
+            "mode": "",
+        }
+
+        # Channel Name
+        channel_display = (
+            self.page.locator("text=Output Channel").locator("..").locator("button")
+        )
+        if channel_display.count() > 0:
+            props["channel"] = channel_display.inner_text().strip()
+
+        # Activation Delay
+        props["activation_delay"] = float(
+            self.console.get_input_field("Activation Delay")
+        )
+
+        # Show Control Chip
+        chip_toggle = (
+            self.page.locator("text=Show Control Chip")
+            .locator("..")
+            .locator("input[type='checkbox']")
+        )
+        if chip_toggle.count() > 0:
+            props["show_control_chip"] = chip_toggle.is_checked()
+
+        # Mode
+        mode_options = ["Fire", "Momentary", "Pulse"]
+        for option in mode_options:
+            try:
+                button = self.page.get_by_text(option).first
+                if button.count() > 0:
+                    class_name = button.get_attribute("class") or ""
+                    if "pluto-btn--filled" in class_name:
+                        props["mode"] = str(option).lower()
+                        break
+            except:
```

don't hide this exception here